OsmAnd/src/net/osmand/plus/quickaction/actions/AddPOIAction.java

```diff
@@ -79,6 +80,29 @@ public AddPOIAction(QuickAction quickAction) {
 		super(quickAction);
 	}
 
+	@Override
+	public int getIconRes(Context context) {
+		if (context instanceof MapActivity) {
+			final OsmandApplication application = (OsmandApplication) (context).getApplicationContext();
+			final MapPoiTypes poiTypes = application.getPoiTypes();
+			final Map<String, PoiType> allTranslatedNames = poiTypes.getAllTranslatedNames(true);
+			PoiCategory category = getCategory(allTranslatedNames);
+
+			if (category != null) {
+
+				category.getIconKeyName();
+
+				String res = category.getIconKeyName();
+				if (res instanceof String && RenderingIcons.containsBigIcon(res)) {
```

Comment:
what for "res instanceof String" ?

Reply:
It's remains from ShowHidePoiAction) with redundant line breaks. Do i need to check res for null?